src/test/components/MarkerChart.test.js

```diff
@@ -97,87 +99,154 @@ function setupWithProfile(profile) {
   };
 }
 
-it('renders MarkerChart correctly', () => {
-  window.devicePixelRatio = 1;
+describe('MarkerChart', function() {
+  it('renders MarkerChart correctly', () => {
+    window.devicePixelRatio = 1;
 
-  const profile = getProfileWithMarkers([...MARKERS, ...NETWORK_MARKERS]);
-  const {
-    flushRafCalls,
-    dispatch,
-    markerChart,
-    flushDrawLog,
-  } = setupWithProfile(profile);
-
-  dispatch(changeSelectedTab('marker-chart'));
-  markerChart.update();
-  flushRafCalls();
-
-  let drawCalls = flushDrawLog();
-  expect(markerChart).toMatchSnapshot();
-  expect(drawCalls).toMatchSnapshot();
+    const profile = getProfileWithMarkers([...MARKERS, ...NETWORK_MARKERS]);
+    const {
+      flushRafCalls,
+      dispatch,
+      markerChart,
+      flushDrawLog,
+    } = setupWithProfile(profile);
 
-  dispatch(changeSelectedTab('network-chart'));
-  markerChart.update();
-  flushRafCalls();
+    dispatch(changeSelectedTab('marker-chart'));
+    markerChart.update();
+    flushRafCalls();
 
-  drawCalls = flushDrawLog();
-  expect(markerChart).toMatchSnapshot();
-  expect(drawCalls).toMatchSnapshot();
+    let drawCalls = flushDrawLog();
+    expect(markerChart).toMatchSnapshot();
+    expect(drawCalls).toMatchSnapshot();
 
-  delete window.devicePixelRatio;
-});
+    dispatch(changeSelectedTab('network-chart'));
+    markerChart.update();
+    flushRafCalls();
 
-it('renders the hoveredItem markers properly', () => {
-  window.devicePixelRatio = 1;
+    drawCalls = flushDrawLog();
+    expect(markerChart).toMatchSnapshot();
+    expect(drawCalls).toMatchSnapshot();
```

Comment:
After I manually ran the tests (after fixing them), I see that these 2 snapshots show an empty network chart panel, could you look why ?
Before your patch we had a `"Load event"` marker for the network chart.
If this is normal, please change the test so that we have one marker drawn here.